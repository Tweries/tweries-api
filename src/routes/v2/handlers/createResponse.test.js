const { matcherV2, reqBase } = require('../../handlers/mocks');
const createResponse = require('./createResponse');

test('response', () => {
  const args = {
    data: { message: "ðŸ‘‹ hello, I'm healty" },
    req: { ...reqBase }
  };

  expect(createResponse(args)).toMatchSnapshot(matcherV2);
});
