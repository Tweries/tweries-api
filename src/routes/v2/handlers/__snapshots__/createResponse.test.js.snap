// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`response 1`] = `
Object {
  "data": Object {
    "message": "ðŸ‘‹ hello, I'm healty",
  },
  "error": undefined,
  "meta": Object {
    "env": "test",
    "name": "tweries-api",
    "timestamp": Any<Number>,
    "version": "0.11.0",
  },
}
`;
